config/routes.rb

```diff
@@ -7,4 +7,9 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  resources :users, only: [:index, :show] do
+      resources :posts, only: [:index]
+  end
+  resources :posts, only: [:index, :show]
+  root "posts#index"
```

Comment:
- [ ] Kindly remove the routes on line 13, as the one above, line 11, is what we need.

- [ ] **[OPTIONAL]** The root route should be `users#index`, not `posts#index`.